ddl/ddl_db_test.go

```diff
@@ -369,6 +369,8 @@ LOOP:
 	// Make sure there is index with name c3_index.
 	c.Assert(nidx, NotNil)
 	c.Assert(nidx.Meta().ID, Greater, int64(0))
+	ctx.Txn().Rollback()
```

Comment:
If don't call rollback here, `c.Assert(ctx.NewTxn(), IsNil)` would fail, at the next line.

That's because  `ctx.NewTxn()` will check `txn != nil` and `txn.Valid()`, if so, `NewTxn()` will commit the txn before create a new one, but commit the txn will get infoschema out of date error.